test/src/test/groovy/org/openremote/test/notification/NotificationTest.groovy

```diff
@@ -581,15 +581,16 @@ class NotificationTest extends Specification implements ManagerContainerTrait {
 
         then: "the email should have been sent to all tenant users"
         conditions.eventually {
-            assert notificationMessages.size() == 3
+            assert notificationMessages.size() == 4
             assert ((EmailNotificationMessage) notificationMessages.get(0)).getText() == "Hello world!"
             assert ((EmailNotificationMessage) notificationMessages.get(0)).getSubject() == "Test"
             assert ((EmailNotificationMessage) notificationMessages.get(1)).getText() == "Hello world!"
             assert ((EmailNotificationMessage) notificationMessages.get(1)).getSubject() == "Test"
-            assert toAddresses.size() == 3
+            assert toAddresses.size() == 4
             assert toAddresses.any { it == "testuser2@openremote.local" }
             assert toAddresses.any { it == "testuser3@openremote.local" }
             assert toAddresses.any { it == "building@openremote.local" }
+            assert toAddresses.any { ((String) it).startsWith("service-account") } //mqtt service account
```

Comment:
What is this?

Reply:
Needed in order to make the test work. Because a Mqtt Keycloak client is added to the demo setup code. Keycloak adds an service user and it seems it also gets an email address.